frontend/src/app/scripts/_components/ScriptInfoBlocks.tsx

```diff
@@ -7,23 +7,38 @@ import {
   CardHeader,
   CardTitle,
 } from "@/components/ui/card";
+import { mostPopularScripts } from "@/config/siteConfig";
 import { extractDate } from "@/lib/time";
-import { Category } from "@/lib/types";
+import { Category, Script } from "@/lib/types";
 import { CalendarPlus } from "lucide-react";
 import Image from "next/image";
 import Link from "next/link";
 import { useMemo, useState } from "react";
 
 const ITEMS_PER_PAGE = 3;
 
+export const getCorrectTypeNaming = (type: string) => {
```

Comment:
```suggestion
export const getDisplayValueFromType = (type: string) => {
```
Perhaps we should create an Enum for the script types, can be done in separate PR though